corescore/mlflowregistry.py

```diff
@@ -8,9 +8,9 @@ class MlflowRegistryError(Exception):
 
 
 class MlflowRegistry(MlflowClient):
-    def __init__(self, client=None, *args, **kwargs):
-        self.client = MlflowClient(client)
-        super().__init__(*args, **kwargs)
+    def __init__(self, registry_uri=None, *args, **kwargs):
+        self.client = MlflowClient(registry_uri)
+        super().__init__(registry_uri=registry_uri, *args, **kwargs)
 
```

Comment:
Ok, as per https://mlflow.org/docs/latest/_modules/mlflow/tracking/client.html#MlflowClient the `registry_uri` if unspecified defaults to the `tracking_uri` which is picked up from the environment's `MLFLOW_TRACKING_URI`, if set. When I've used this I've never passed in a URI explicitly, so not stumbled here.

We shouldn't need to both inherit from MLFlowClient and also create and store one internally? It would be simpler to only do the former (and not pass around the URIs). I can try making that change...

Reply:
Added a commit as above. then remembered to hit 'Submit Review' afterwards. Otherwise it looks good to me and will be useful in the mob next week.

Reply:
As discussed, the changes are good. Thank you. Merging.